### ISSUE TITLE
change background of frame

### ISSUE THREAD

**Issue by RityZhang** [UCSB-CS56-Projects/cs56-networking-chat#43]
Add the button to change the bg of frame

**Comment by jmangel** [comment]
This could also be a dropdown menu, like changing font is now (thanks to you). Let's say 100-150 points based on button/dropdown

**Comment by RityZhang** [comment]
Yes. What I think is to create a dropdown so that user can select.

**Comment by seemantasaha** [comment]
W18 OK. 100 points.
